CuriousKatie/ViewController.swift

```diff
@@ -9,10 +9,24 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        /// Creates a list of random people.
+        let listOfPeople = Helper.pickSomePeople()
+        
+        /// Participants introduce themselves
+        for person in listOfPeople {
+            print(person.introduceMyself())
+        }
+        
+        /// Print a new line to separate sections.
+        print("\n")
+        
+        /// All participants share all of their interests.
+        for person in listOfPeople {
+            print(person.shareMyInterests())
```

Comment:
Are you not supposed to log **one** unshared interest **at a time**?

Reply:
I just realized that you were planing on implementing this later as part of #6, you can ignore the previous comment.

Reply:
Yes, ok 🙃This was just to check that I'm actually creating random interests for each person.